packages/apollo/src/commands/service/check.ts

```diff
@@ -27,6 +30,73 @@ const formatChange = (change: Change) => {
   };
 };
 
+interface TasksOutput {
+  config: ApolloConfig;
+  gitContext?: GitContext;
+  checkSchemaResult: CheckSchema_service_checkSchema;
+  shouldOutputJson: boolean;
+  shouldOutputMarkdown: boolean;
+}
+
+export function formatMarkdown({
+  checkSchemaResult,
+  serviceName,
+  tag
+}: {
+  checkSchemaResult: CheckSchema_service_checkSchema;
+  serviceName: string;
+  tag: string;
+}): string {
+  const { diffToPrevious } = checkSchemaResult;
+
+  if (!diffToPrevious) {
+    throw new Error("checkSchemaResult.diffToPrevious missing");
+  }
+
+  const { validationConfig } = diffToPrevious;
+
+  if (!validationConfig) {
+    throw new Error(
+      "checkSchemaResult.diffToPrevious.validationConfig missing"
+    );
+  }
+
+  // This will always return a negative number of days. Use `-` to make it positive.
+  const days = -moment()
```

Comment:
I found the formatting a bit hard to understand. Can we use [`Math.abs()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs)?

Reply:
Yes! That would be much more legible 👍